src/types.ts

```diff
@@ -1,17 +1,12 @@
 import { DataSourceSettings, SelectableValue } from '@grafana/data';
 import { AwsAuthDataSourceJsonData, AwsAuthDataSourceSecureJsonData, SQLQuery } from '@grafana/aws-sdk';
+import { FillValueOptions } from '@grafana/aws-sdk/dist/sql/QueryEditor/FillValueSelect';
```

Comment:
if you increase the version of the sdk and merge it, a new release is automatically generated so you could use the nice import path here

Reply:
https://github.com/grafana/grafana-aws-sdk-react/pull/11

It doesn't seem to build with that explicit path, by the way. Would you happen to know why?

Reply:
looks like some misconfiguration of jest: https://stackoverflow.com/questions/50863312/jest-gives-cannot-find-module-when-importing-components-with-absolute-paths

It should go away with the changes in the new version.

BTW, you also need to update `src/QueryEditor.test.tsx` (FillValueOptions is also used there)